webapp/src/components/mobilitymap/TransactionCard.js

```diff
@@ -0,0 +1,37 @@
+import FormError from "../FormError";
+import i18next from "i18next";
+import React from "react";
+import { Button, Card } from "react-bootstrap";
+
+const TransactionCard = ({ endTrip, onCloseTrip, error }) => {
+  const { rate, provider, id } = endTrip;
+  const { localizationVars: locVars } = rate;
+  const handleClose = () => onCloseTrip();
+  return (
+    <Card className="cardContainer">
+      <Card.Body>
+        {!error ? (
+          <>
+            <p>
+              Trip {id} with {provider.vendorName} has ended.
+            </p>
+            <p>
+              {i18next.t(rate.localizationKey, {
+                surchargeCents: locVars.surchargeCents * 0.01,
+                unitCents: locVars.unitCents * 0.01,
+                ns: "mobility",
+              })}
+            </p>
+            <Button size="sm" variant="secondary" className="w-100" onClick={handleClose}>
+              Close
+            </Button>
+          </>
+        ) : (
+          <FormError error={error} />
```

Comment:
`FormError` should not be conditionally rendered- it manages hiding itself so it can transition (grow/shrink/fade in/out). If you remove it from the DOM, it just pops in and can't transition.